ci/publish-crate.sh

```diff
@@ -35,8 +43,22 @@ for Cargo_toml in $Cargo_tomls; do
     # so use the solana rust docker image until this is resolved upstream
     source ci/rust-version.sh
     ci/docker-run.sh "$rust_stable_docker_image" bash -exc "cd $crate; $cargoCommand"
-    #ci/docker-run.sh rust bash -exc "cd $crate; $cargoCommand"
   ) || true # <-- Don't fail.  We want to be able to retry the job in cases when a publish fails halfway due to network/cloud issues
+
+  crate_name=$(grep -m 1 '^name = ' $Cargo_toml | cut -f 3 -d ' ' | tr -d \")
+  numRetries=30
+  sleepTime=2
```

Comment:
I'd just inline the 2 in the sleep below . `sleepTime=2` doesn't really add much since it's only used once